testing/e2e.sh

```diff
@@ -69,54 +63,15 @@ EOF
   rm -f output.txt
 }
 
-function test_launcher_image() {
-  cd "${ROOT}"
-  clear_docker
-  EXPECT_CONTAINS "$(bazel run "$@" testdata:launcher_image)" "Launched via launcher!"
-}
-
 function test_java_image() {
   cd "${ROOT}"
-  clear_docker
+  clear_docker_full
```

Comment:
Why not use the new clear_docker everywhere instead of using clear_docker_full at all?

Reply:
Don't want to mess up with the existing e2e.sh functionality

Reply:
sgtm